packages/app/src/app/overmind/namespaces/editor/internalActions.ts

```diff
@@ -186,7 +186,6 @@ export const saveCode = async (
 
     if (savedCode === null) {
       // If the savedCode is also module.code
-      effects.moduleRecover.remove(sandbox.id, module);
```

Comment:
Not sure if we need to do anything else regarding `syncModule` if we've remove the module recovery functionality from this if block.